frontend/src/plugins/__tests__/auth.spec.js

```diff
@@ -8,9 +8,28 @@ Vue.use(storeLoader)
 // expired on 01-01-1970
 const expiredJWTPayload =
   'eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpYXQiOjE2MzMxMzM0MDksImV4cCI6MCwicm9sZXMiOlsiUk9MRV9VU0VSIl0sInVzZXJuYW1lIjoidGVzdC11c2VyIiwidXNlciI6Ii91c2Vycy8xYTJiM2M0ZCJ9'
+// {
```

Comment:
minor:
i would use a multi line comment for multi line texts.
This makes copy/paste easier.